### DISCUSSION TITLE
Re-publish a github-pages

### DISCUSSION THREAD

**Discussion by maiCay** [community/community#36713] [Other Feature Feedback, Questions, & Ideas]
Hi! I would like to know if it's possible to undo an unpublish! We made a mistake! And we want our page back! But we don't know how to do that!

**Comment by radar-blip** [comment]
You want to unpublish a Github Page? 
There's a way to do that, go to your repository > Settings > Pages, and on the right there's an option to unpublish.

![Screenshot from 2022-10-19 18-00-53](https://user-images.githubusercontent.com/39196767/196813955-3e8fe8a2-40eb-4f68-bd28-26d11be1e785.png)

Sorry if I misunderstood your question.

**Reply by maiCay** [reply]
I want to revert this action! We unpublish the page because the domain doesn't work well, and now we´re trying to have a github-page again! But it's not posible to deploy a page again from main.

**Reply by radar-blip** [reply]
I'm sure you have looked in your Repository Settings, under Pages, and check here:

![deploy](https://user-images.githubusercontent.com/39196767/196821300-176c2fec-427a-4c92-b4a4-54ea131a71fa.png)

You should be able to just re-deploy the page (from main).

**Reply by maiCay** [reply]
Thanks!! It works!!

**Reply by radar-blip** [reply]
You're welcome. Glad I could help!

**Comment by ossanna16** [comment]
Hey there @maiCay and thank you for asking a good question! I'm glad you found our community 🙂
